src/components/BackToTop.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div :class="{ show: visible }" class="scroll-to-top" @click="backToTop">
+    <img src="@/assets/up-arrow.png" alt="arrow" width="50px" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ScrollToTop",
```

Comment:
If you are using the lib, we no longer need this component, I believe.

Delete it.